### PR TITLE
[#IP-85] Expose IDP list through public endpoint

### PR DESCRIPTION
**Pull request by fabriziopapi**
Add 2 resource locations for spid ip providers list and images (.png).

### CODE REVIEW

**Review by balanza — APPROVED** [review]
LGTM, wait for https://github.com/pagopa/io-infrastructure-live-new/pull/482 before merging

File: proxies.json [hunk 1]
```diff
@@ -70,6 +70,20 @@
...
+        "route": "/spid/idps/list.json"
+      },
+      "backendUri": "https://%STATIC_WEB_ASSETS_ENDPOINT%/spid/idps/list.json"
+    },
+    "idpslogo":{
+      "matchCondition": {
+        "methods": ["GET"],
+        "route": "/spid/idps/{idp}.png"
+      },
+      "backendUri": "https://%STATIC_WEB_ASSETS_ENDPOINT%/spid/idps/{idp}.png"
```
Comment (pasqualedevita) [inline proxies.json:86]:
```suggestion
    },
    "spidIdp":{
      "matchCondition": {
        "methods": ["GET"],
        "route": "/spid/idps/{*path}"
      },
      "backendUri": "https://%STATIC_WEB_ASSETS_ENDPOINT%/spid/idps/{path}"
    },
```

File: proxies.json [hunk 2]
```diff
@@ -70,6 +70,20 @@
...
       "backendUri": "https://%STATIC_WEB_ASSETS_ENDPOINT%/logos/privative/{path}"
+    },
+    "idpslist":{
+      "matchCondition": {
+        "methods": ["GET"],
+        "route": "/spid/idps/list.json"
+      },
+      "backendUri": "https://%STATIC_WEB_ASSETS_ENDPOINT%/spid/idps/list.json"
+    },
+    "idpslogo":{
```
Comment (pasqualedevita) [inline proxies.json:81]:
```suggestion
    "idpsLogos":{
```
camel case

File: proxies.json [hunk 3]
```diff
@@ -70,6 +70,20 @@
         "route": "/logos/privative/{*path}"
       },
       "backendUri": "https://%STATIC_WEB_ASSETS_ENDPOINT%/logos/privative/{path}"
+    },
+    "idpslist":{
```
Comment (pasqualedevita) [inline proxies.json:74]:
```suggestion
    "idpsList":{
```
camel case

**Review by balanza — APPROVED** [review]

**Review by pasqualedevita — APPROVED** [review]
@balanza @fabriziopapi I commit a fix on brackets and lint. LGTM

**Review by balanza — APPROVED** [review]

**Pull request merged**